scripts/kata-arch.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+typeset -r script_name=${0##*/}
+
+die()
```

Comment:
Consider pulling this into a lib.sh like we do in many other places - I know this is the first and only script in this dir, but if we do it now then the changes of somebody just doing a clone later without the split are vastly reduced :-)

Reply:
Fixed.